pkg/controller/job/job_controller.go

```diff
@@ -916,6 +927,7 @@ func (jm *Controller) syncJob(ctx context.Context, key string) (rErr error) {
 			jobCtx.finishedCondition = newCondition(batch.JobFailureTarget, v1.ConditionTrue, batch.JobReasonPodFailurePolicy, *failJobMessage, jm.clock.Now())
 		}
 	}
+	// 2.2. Evaluate any other negative criteria.
```

Comment:
Be specific here about what those criteria are, here backoffLimitExceeded, activeDeadlineExceeded.

Reply:
That makes sense, thanks!

Reply:
Done.